test/gui/shared/scripts/helpers/api/oc10.py

```diff
@@ -39,3 +39,37 @@ def add_user_to_group(user, group_name):
     body = {"groupid": group_name}
     response = request.post(url, body)
     request.assertHttpStatus(response, 200)
+
+
+def create_user(username, password, displayname, email):
+    url = url_join(get_ocs_url(), "users")
+    body = {
+        "userid": username,
+        "password": password,
+        "displayname": displayname,
+        "email": email,
+    }
+    response = request.post(url, body)
+    request.assertHttpStatus(response, 200)
+    # oc10 does not set display name and email while creating user,
+    # so we need update the user info
+    user_url = url_join(get_ocs_url(), "users", username)
+    display_name_body = {"key": "displayname", "value": displayname}
+    display_name_response = request.put(user_url, display_name_body)
+    request.assertHttpStatus(display_name_response, 200)
+    # email_body = {"key": "email", "value": email}
+    # email_response = request.put(user_url, email_body)
+    # request.assertHttpStatus(email_response, 200)
```

Comment:
?

Reply:
email is not needed, so i think it's better to remove